docs/wcs/wcsapi.rst

```diff
@@ -287,20 +287,22 @@ If you are implementing your own WCS class, you could choose to implement
 :class:`~astropy.wcs.WCS` class does this - the example above can be written
 more succinctly as::
 
-    >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA
+    >>> wcs[10, 30:100, 30:100]  # doctest: +REMOTE_DATA +ELLIPSIS
+    <BLANKLINE>
```

Comment:
Makes no sense but I'll take the green status. Thank you so much, @bsipocz ! I can't approve my own PR, so feel free to approve it and merge. 🙏

Reply:
I don't think that an ELLIPSIS can start a block, but this is just a wild guess. I tried the permutations until it worked 🤷‍♀

Reply:
Thank you!!!